<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCValueCard</class>
 <widget class="QFrame" name="NewCValueCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>储值卡</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonBigBG-normal1366x768.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>1231</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QTableWidget" name="tableValue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>146</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <column>
     <property name="text">
      <string>卡号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>入场必须刷卡</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出场必须刷卡</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡余额</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>扣费标准</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡片自编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>备注</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>登记者</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>1231</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>业主信息</string>
     </attribute>
     <widget class="QGroupBox" name="gbOwner">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1201</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>资料</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>391</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>单位：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtUnit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>391</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtAddr"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>391</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>行驶证号码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtDriveID"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>235</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string>业主照片</string>
       </property>
       <widget class="QLabel" name="lblOwner">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>214</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>30</y>
         <width>351</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string>车辆照片</string>
       </property>
       <widget class="QLabel" name="lblCar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>321</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="lblUserID">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>40</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lblCardNo">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_14">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>90</y>
         <width>391</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>证件号码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtID"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>391</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>电话：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtTel"/>
        </item>
       </layout>
      </widget>
      <widget class="QComboBox" name="cbxCarType">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>211</y>
         <width>149</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>211</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbxSex">
       <property name="geometry">
        <rect>
         <x>339</x>
         <y>31</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>31</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtName">
       <property name="geometry">
        <rect>
         <x>53</x>
         <y>31</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>361</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>361</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号9：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID0">
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>241</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>241</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号0：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>331</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>331</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号8：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>301</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号7：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>301</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>271</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>271</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号6：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>241</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>241</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号5：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>301</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>301</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号2：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>361</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号4：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>361</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>331</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>331</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号3：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>271</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号1：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtVechileID1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>271</y>
         <width>129</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnValueSave">
       <property name="geometry">
        <rect>
         <x>1120</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image:url(none);</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnValueRecharge">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image:url(none);</string>
       </property>
       <property name="text">
        <string>充值卡充值</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>续费纪录</string>
     </attribute>
     <widget class="QTableWidget" name="tableRecharge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1201</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <column>
       <property name="text">
        <string>收费金额</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时长</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>收费时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>费用种类</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>收费人员</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>最近进出记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableAccess">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>进入通道</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>进入时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>离开通道</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>离开时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>记录ID</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>1</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>图像1</string>
      </property>
      <widget class="QLabel" name="lblImg1">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>0</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>图像2</string>
      </property>
      <widget class="QLabel" name="lblImg2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>200</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>图像3</string>
      </property>
      <widget class="QLabel" name="lblImg3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>200</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>图像4</string>
      </property>
      <widget class="QLabel" name="lblImg4">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>出入权限</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableEntranceRight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1201</width>
        <height>361</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>通道名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>起始时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnEntranceSave">
      <property name="geometry">
       <rect>
        <x>1330</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveRight">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMinimalize">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>587</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>4</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
